include/cbor_encode.h

```diff
@@ -72,7 +72,7 @@ bool list_end_encode(cbor_state_t *state, size_t max_num);
 bool map_end_encode(cbor_state_t *state, size_t max_num);
 
 /** Encode a "nil" primitive value. result should be NULL. */
```

Comment:
Comment should be updated for new verb use.

Reply:
I don't use 'put' in the other docstrings since it's all encoding. The difference is just the type of the argument and I needed another verb to distinguish. So I don't really want to change it.

The reason I changed this now is that it makes the code generation slightly simpler.